### PR TITLE
update server installation docs

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Ticket: [MEN-5973](https://tracker.mender.io/browse/MEN-5973)

### CODE REVIEW

File: 07.Server-installation/04.Production-installation-with-kubernetes/docs.md [hunk 1]
```diff
@@ -59,7 +60,7 @@ bucket or install and expose a storage service providing S3-compatible APIs over
 
 The following open-source projects are compatible with the S3 APIs:
 
-* [Minio](https://github.com/chrislusf/seaweedfs)
+* [Minio](https://github.com/minio/minio)
```
Comment (lluiscampos) [inline 07.Server-installation/04.Production-installation-with-kubernetes/docs.md:63]: Nice catch, thank you.

**Comment by kjaskiewiczz** [comment]
It doesn't feel like it's well documented, but I didn't have a better idea, sorry;
all suggestions very welcome! thanks in advance!

File: 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: OpenSearch
+taxonomy:
+    category: docs
+    label: tutorial
+---
+
+The Mender server uses OpenSearch as storage and search engine.
```
Comment (tranchitella) [inline 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md:8]:
```suggestion
The Mender server uses OpenSearch as a storage and search engine.
```

File: 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+To install OpenSearch on Kubernetes cluster using the Helm chart follow instructions from [Opensearch using `helm`](https://opensearch.org/docs/2.0/opensearch/install/helm/):
```
Comment (tranchitella) [inline 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md:10]:
```suggestion
To install OpenSearch on a Kubernetes cluster using the Helm chart follow instructions from [Opensearch using `helm`](https://opensearch.org/docs/2.0/opensearch/install/helm/):
```

**Review by tranchitella — APPROVED** [review]

File: 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md (hunk 3 above)
Comment (oldgiova) [reply 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md:10]: We're using opensearch 2.4 version:
```suggestion
To install OpenSearch on a Kubernetes cluster using the Helm chart follow instructions from [Opensearch using `helm`](https://opensearch.org/docs/2.4/install-and-configure/install-opensearch/helm/):
```

File: 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md (hunk 2 above)
Comment (kjaskiewiczz) [reply 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md]: fixed

File: 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md (hunk 3 above)
Comment (kjaskiewiczz) [reply 07.Server-installation/04.Production-installation-with-kubernetes/07.OpenSearch/docs.md]: fixed

**Pull request merged**